Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -172,6 +173,20 @@ class ThumbnailCell: UICollectionViewCell {
         return backgroundImage
     }()
 
+    lazy var selectedOverlay: UIView = {
+        let selectedOverlay = UIView()
+        selectedOverlay.backgroundColor = ThumbnailCellUX.SelectedOverlayColor
+        selectedOverlay.hidden = true
+        return selectedOverlay
+    }()
+
+    override var selected: Bool {
+        didSet {
+            super.selected = selected
```

Comment:
Nit: Don't think you need this line since when didSet is called, selected is already set.